### ISSUE TITLE
Allow per-player i18n for command help

### ISSUE THREAD

**Issue by XHawk87** [JorelAli/CommandAPI#528]
### Description

I'd like to be able to supply help messages to users for commands in the user's preferred language.

### Expected code

```
new CommandAPICommand("blah")
  .withHelp(makeHelpTopic("blah"))
  .executes( ... )
  .register();
```

```
public HelpTopic makeHelpTopic(String commandName) {
  return new HelpTopic() {
    @Override
    public String getFullText(@NotNull CommandSender forWho) {
      // Something something i18n
    }
  }
}
```
This would be an alternative to `.withHelp(String shortDescription, String fullDescription)`

**Comment by JorelAli** [comment]
Implemented for 9.4.0

**Comment by JorelAli** [comment]
Implemented in version 9.4.0.
